client/src/components/Navbar/Navbar.css

```diff
@@ -1,4 +1,133 @@
+a:hover, a:visited, a:link, a:active{
+   text-decoration: none;
+   color: inherit;
+}
+
+.bg-nav{
+      opacity: 0.8;
+}
+
 .navbar-component .active {
-    font-weight: 900;
-    color: black;
-}
\ No newline at end of file
+      font-size: 1.2rem;
+      font-weight: 900;
+      color: white;
+}
+.navbar{
+      background-color: #080808;
+      padding: 3px 15px 3px 10px;
+}
+#navLink{
+      font-weight: 400;
+      font-style: italic;
+      color: #fff;
+}
+
+
+.nav-item{
+     color:#fff;
+     margin: 10px 25px;
+}
+
+.nav-logo{
+      opacity: 1;
+      height: 45px;
+      margin-left: 40px;
+}
+
+.navbar-toggler{
+      background-color: #7e8a97;
+}
+
+.search-bar{
+      border-radius:35px;
+      width:180px!important;
+      height: 30px;
+      margin-right: 25px;
+      margin-left: 20px;
+      padding-left: 35px;
+      }
```

Comment:
Stick the closing parenthesis to the margin here.

Reply:
Done